packages/vbet/vbet/vbet.py

```diff
@@ -84,11 +85,20 @@ def vbet(huc, flowlines, flowareas, orig_slope, max_slope, orig_hand, hillshade,
     vbet_network(flowlines, flowareas, cfg.OUTPUT_EPSG, vbet_network_path)
     project.add_project_vector(proj_nodes['Intermediates'], LayerTypes['VBET_NETWORK'])
 
+    # Get raster resolution as min buffer and apply bankfull width buffer to reaches
+    with rasterio.open(proj_slope) as raster:
+        t = raster.transform
+        min_buffer = (t[0] + (-t[4])) / 2
```

Comment:
Cell height is **nearly** always negative, but there are exceptions. I would prefer to see a root of the sum of the squares approach to calculating cell resolution. Also I think we have code for this in our raster code in rscommons.

Reply:
@KellyMWhitehead I could be wrong but I think pixel height is the 5th element... starring Bruce Willis

Also I think this is as easy as wrapping in abs()

```python
            # Remember:
            # [0]/* top left x */
            # [1]/* w-e pixel resolution */
            # [2]/* rotation, 0 if image is "north up" */
            # [3]/* top left y */
            # [4]/* rotation, 0 if image is "north up" */
            # [5]/* n-s pixel resolution */
            min_buffer = (t[0] + abs(t[4])) / 2
```

Reply:
Ignore this. GDAL seems to use a different order for that array than Rasterio.... good to know.